### ISSUE TITLE
Syri failed to run. Procedure

### ISSUE THREAD

**Issue by hgz2021** [schneebergerlab/syri#138]
Hello, I made a mistake running syir and need your help.
I installed Syri using Conda, and here is my command
Conda create -n syri
conda activate syri
cd syri
conda install -c bioconda syri
But I had a problem running the sample。 I ran the command syri
minimap2 -ax asm5 -t 4 --eqx A.fa B.fa \
 | samtools sort -O BAM - > A_B.bam
syri -c A_B.bam -r A.fa -q B.fa -F B --prefix A_B
finally,the picture below is an error
![1d64e52c8ff33d013d88a3c56fcf24b](https://user-images.githubusercontent.com/87291139/165667215-0df7eeac-030c-4417-9fe3-9d5e2a6a877b.png)

**Comment by mnshgl0110** [comment]
The last line in the pic says that it is trying to access file "/home/....//home/../A_Bmapids.txt". Based on other paths, I assume that this is wrong. Though it is not clear why this is happening. A would suggest that you retry in an empty folder containing only the genomes and the alignment file.

**Comment by hgz2021** [comment]
They were in a folder.
![image](https://user-images.githubusercontent.com/87291139/165718992-09280ae7-c7e0-45bc-a1cd-886c36a0152b.png)

**Comment by hgz2021** [comment]
Hello, the last problem was solved when I changed the chromosomes of 'ref' and 'qry' to 'chr1,chr2.....chr* *'. But when I run it with my own data, a new error occurs.
The corresponding error message is reported.
![image](https://user-images.githubusercontent.com/87291139/166873559-e4d7b71f-29f9-4e6a-8110-8eb5bcfdc095.png)

syri.log
![image](https://user-images.githubusercontent.com/87291139/166873651-42c7fc3e-dce7-4654-83f2-b95c1bf66733.png)

**Comment by hgz2021** [comment]
Hello, the last problem was solved when I changed the chromosomes of 'ref' and 'qry' to 'chr1,chr2.....chr* *'. But when I run it with my own data, a new error occurs.
The corresponding error message is reported.
![image](https://user-images.githubusercontent.com/87291139/166873559-e4d7b71f-29f9-4e6a-8110-8eb5bcfdc095.png)

syri.log
![image](https://user-images.githubusercontent.com/87291139/166873651-42c7fc3e-dce7-4654-83f2-b95c1bf66733.png)
